Automattic/jetpack
# My Jetpack: Fix Jetpack AI bi-yearly plan on product card

Fixes #34275

## Proposed changes:

* Adds the Jetpack AI Bi-yearly plan to the product check

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to My Jetpack without a Jetpack AI plan
* Buy the bi-yearly plan
* Go back to My Jetpack
* Check that the Jetpack AI card shows as active with a disabled "Manage" button

| Before | After
| - | -
| ![2023-11-23_15-33-08](https://github.com/Automattic/jetpack/assets/8486249/7bc85ab7-a027-4343-969e-ab51613f7c64) | ![2023-11-23_16-06-43](https://github.com/Automattic/jetpack/assets/8486249/cbc4ca36-2fcd-41b0-9348-4888bd527137)
